### PR TITLE
Fix table in DNS Report

### LINKED ISSUE #3635: DNS report - Wrong data in 'other findings found' table
**Describe the bug**
When creating a DNS Report, the table in section "Other findings found" shows weird data.
Also, when expanding the table row, this whole dict is shown instead of a finding name.
Besides this, the occurrences section seems to be wrong as well, since it is repeating the sentence multiple times.

**To Reproduce**

1. Create a DNS Report
2. Check the section "Other findings found"
3. Expand the table row

**Expected behavior**
- Show finding name in table
- Show finding name in title when expanding the row
- Show "This nameserver has no IPv6 address" only once

**Screenshots**
Image

### PR DESCRIPTION
**Pull request by madelondohmen**
### Changes
This PR fixes the "Other findings found" table in the DNS Report.

### Issue link
Closes #3635

### Demo
afbeelding

### QA notes
- Create a DNS Report
- Check the chapter "Other findings found"
- Findings in table should be readable, all occurences should be visible with title and description

---

### Code Checklist

- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] I have checked the documentation and made changes where necessary.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

- [ ] Tickets have been created for newly discovered issues.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — APPROVED** [review]

**Review by ammar92 — APPROVED** [review]

**Comment by stephanie0x00** [comment]
## Checklist for QA:

- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
  - [x] I have managed to pass the onboarding flow
  - [x] Objects and Findings are created properly
  - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.
- [x] I checked the logs for errors and/or warnings and made issues where necessary

### What works:
Looks good. The section only shows up when having one of the following findings: 
-  "KAT-INVALID-SPF"
- "KAT-NAMESERVER-NO-IPV6" 
-  "KAT-NAMESERVER-NO-TWO-IPV6"

### What doesn't work:
n/a

### Bug or feature?:
n/a

**Comment by underdarknl** [comment]
Do we not have a description available for these findings? Description None for no ipv6 address seems illogical.

**Comment by underdarknl** [comment]
We do:
https://github.com/minvws/nl-kat-coordination/blob/1894cd9f96e096c520df6a3e069a6aef989db8d9/boefjes/boefjes/plugins/kat_kat_finding_types/kat_finding_types.json#L157
that means its somehow not correctly loaded in this report

**Comment by underdarknl** [comment]
Possibly because the Finding is still Pending, and as such has not been hydrated from the linked file. Does it function properly when the finding is no longer pending?

**Comment by madelondohmen** [comment]
> Possibly because the Finding is still Pending, and as such has not been hydrated from the linked file. Does it function properly when the finding is no longer pending?

The Finding has been deleted (I guess when it was no longer pending). But I've added it manually now and then it **does** correctly show a description: 

afbeelding

**Comment by stephanie0x00** [comment]
It now looks like this. So can me merged.

image

File: rocky/reports/report_types/dns_report/report.html [hunk 1]
```diff
@@ -153,13 +153,12 @@ <h3>{% translate "Other findings found" %}</h3>
...
                     <tr class="expando-row">
                         <td colspan="5">
-                            <h3>{{ info.finding_type }}</h3>
                             <h4>{% translate "Findings information" %}</h4>
                             <dl>
                                 <div>
                                     <dt>{% translate "Finding" %}</dt>
                                     <dd>
-                                        {{ info.finding_type.id }}
+                                        {{ info.finding_type.primary_key }}
```
Comment (underdarknl) [inline rocky/reports/report_types/dns_report/report.html:161]: Should we not show the human readable here too?

Comment (madelondohmen) [reply rocky/reports/report_types/dns_report/report.html:161]: Would be nice, but this isn't possible here

**Review by underdarknl — APPROVED** [review]

**Pull request merged**